pkg/reconciler/pubsub/reconciler.go

```diff
@@ -19,13 +19,15 @@ package pubsub
 import (
 	"context"
 	"fmt"
+	"github.com/google/knative-gcp/pkg/apis/duck/v1alpha1"
 
 	pubsubsourcev1alpha1 "github.com/google/knative-gcp/pkg/apis/pubsub/v1alpha1"
```

Comment:
pubsubv1alpha1 alias might be more appropriate

Reply:
I checked the codebase. We usually use the alias "duckv1alpha1"